Source/ReadyPlayerMe/Private/Request/ReadyPlayerMeBaseRequest.cpp

```diff
@@ -19,6 +21,11 @@ namespace
 	template <typename RequestPtr>
 	void AddRPMHeaders(RequestPtr HttpRequest)
 	{
+		const UReadyPlayerMeSettings* Settings = GetDefault<UReadyPlayerMeSettings>();
+		if (Settings)
+		{
+			HttpRequest->SetHeader(HEADER_APP_ID, Settings->AppId);
```

Comment:
Just as an info info to @MaxAndreassenRPM we will always send the header, even if it's empty.

Reply:
No problem for the backend as it'll just be disregarded if it's empty 👍